package.json

```diff
@@ -65,7 +65,6 @@
   "dependencies": {
     "broccoli-funnel": "^1.0.1",
     "ember-cli-babel": "^5.1.6",
-    "ember-getowner-polyfill": "^1.0.0",
```

Comment:
You can leave this in here. The deprecation warning is importing `getOwner` instead of destructuring it.